optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -900,14 +901,18 @@ def _save_pretrained(
                 The decoder with past key values model file name overwriting the default file name, allowing to save
                 the decoder model with a different name.
         """
-        src_file_names = [self.encoder_model_path, self.decoder_model_path]
-        dst_file_names = [encoder_file_name, decoder_file_name]
+        src_paths = [self.encoder_model_path, self.decoder_model_path]
+        dst_file_names = ["encoder_model/" + encoder_file_name, "decoder_model/" + decoder_file_name]
         if self.use_cache:
-            src_file_names.append(self.decoder_with_past_model_path)
-            dst_file_names.append(decoder_with_past_file_name)
+            src_paths.append(self.decoder_with_past_model_path)
+            dst_file_names.append("decoder_with_past_model/" + decoder_with_past_file_name)
```

Comment:
Why not use path join instead?

Reply:
Because `dst_file_names` were strings initially. It's true that it would be more proper to have them as Paths, but then I would have to make sure everything else still works